packages/lib/webpack.config.js

```diff
@@ -135,4 +137,51 @@ const configure = (prod) => ({
   },
 });
 
-module.exports = (env, argv) => configure(argv.mode === 'production');
+const configureLib = (prod) => ({
+  name: 'lib',
+  externals: {
+    three: 'three',
+    lodash: 'lodash',
+  },
+  entry: path.resolve(__dirname, 'src', 'index.js'),
+  plugins: [
+    new webpack.DefinePlugin({
+      PACKAGE_VERSION: JSON.stringify(version.combined),
+      DEBUG: !prod,
+    }),
+  ],
+  module: {
+    rules: [{
+      test: /\.(vert|frag)$/,
+      use: 'raw-loader',
+    }, {
+      test: /\.js$/,
+      exclude: [
+        /(node_modules[\\/](?!three))|vendor/,
+        path.resolve(__dirname, 'src/utils/SelectionParser.js'),
+        path.resolve(__dirname, 'src/utils/MiewCLIParser.js')],
+      use: {
+        loader: 'babel-loader',
+        options: {
+          presets: ['@babel/preset-env'],
+        },
+      },
+    },
+    ],
+  },
+  output: {
+    path: path.resolve(__dirname, 'build'),
+    filename: '${packageJson.main}',
+    chunkFilename: '[name].[chunkhash].js',
+    library: 'Miew',
+    libraryTarget: 'umd',
+    umdNamedDefine: true,
+    globalObject: 'typeof self !== "undefined" ? self : this',
```

Comment:
Magic? 😁

Reply:
Perhaps, `this` is enough? https://webpack.js.org/configuration/output/#outputglobalobject
I'm not sure though.